cob_cartesian_controller/include/cob_cartesian_controller/cartesian_controller_data_types.h

```diff
@@ -37,16 +37,23 @@ namespace cob_cartesian_controller
 {
     struct ProfileStruct
     {
+        double t_ipo;
         unsigned int profile_type;
         double vel, accl;
+        double Se_max;
+    };
+
+    struct ProfileTimings
+    {
+        double tb, te, tv;
+        unsigned int steps_tb, steps_te, steps_tv;
+        bool ok;
```

Comment:
could we get rid of this?
E.g. by letting the according `getProfileTimings` function return a bool instead of a ProfileTimings struct. The struct should then be passed to the function as a reference parameter (also see my comment below!)